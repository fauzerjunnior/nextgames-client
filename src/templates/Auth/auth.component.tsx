import * as S from './auth.styles';

const Auth = () => (
  <S.Wrapper>
    <h1>Auth</h1>
  </S.Wrapper>
);

export default Auth;
