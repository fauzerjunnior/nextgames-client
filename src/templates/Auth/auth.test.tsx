import { render, screen } from '@testing-library/react';
import Auth from './auth.component';

describe('Auth', () => {
  it('should render the Auth', () => {
    const { container } = render(<Auth />);

    expect(screen.getByRole('heading', { name: /Auth/i })).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
