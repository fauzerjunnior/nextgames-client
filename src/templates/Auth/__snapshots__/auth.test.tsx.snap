// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth should render the Auth 1`] = `
<main
  class=""
>
  <h1>
    Auth
  </h1>
</main>
`;
